mws/resource_mws_pas_test.go

```diff
@@ -0,0 +1,246 @@
+package mws
+
+import (
+	"context"
+	"testing"
+
+	"github.com/databrickslabs/databricks-terraform/common"
+	"github.com/databrickslabs/databricks-terraform/internal/qa"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMWSPAS(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test in short mode.")
+	}
+	acctID := qa.GetEnvOrSkipTest(t, "DATABRICKS_ACCOUNT_ID")
+	client := common.CommonEnvironmentClient()
+	ctx := context.Background()
+	pasAPI := NewPrivateAccessSettingsAPI(ctx, client)
+	pasList, err := pasAPI.List(acctID)
+	assert.NoError(t, err, err)
+	t.Log(pasList)
+
+	pas := PrivateAccessSettings{
+		AccountID: acctID,
+		PasName:   qa.RandomName(),
+		PasID:     "",
+	}
+	err = pasAPI.Create(&pas)
+	assert.NoError(t, err, err)
+	defer func() {
+		err = pasAPI.Delete(acctID, pas.PasID)
+		assert.NoError(t, err, err)
+	}()
+
+	myPAS, err := pasAPI.Read(acctID, pas.PasID)
+	assert.NoError(t, err, err)
+	t.Log(myPAS)
+}
+
+func TestResourcePASCreate(t *testing.T) {
+	d, err := qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
+			{
+				Method:   "POST",
+				Resource: "/api/2.0/accounts/abc/private-access-settings",
+				ExpectedRequest: PrivateAccessSettings{
+					AccountID: "abc",
+					Region:    "ar",
+					PasName:   "pas_name",
+					PasID:     "pas_id",
+				},
+			},
+			{
+				Method:   "GET",
+				Resource: "/api/2.0/accounts/abc/private-access-settings/pas_id",
+
+				Response: PrivateAccessSettings{
+					AccountID: "abc",
+					Region:    "ar",
+					PasName:   "pas_name",
+					PasID:     "pas_id",
+				},
+			},
+		},
+		Resource: ResourcePrivateAccessSettings(),
+		HCL: `
+		account_id = "abc"
+		private_access_settings_name = "pas_name"
+		region = "ar"
+		private_access_settings_id = "pas_id"
```

Comment:
`private_access_settings_id` is computed and therefore should not be specified in HCL

Reply:
Removed